lib/taskana-core/src/test/java/pro/taskana/security/JaasExtension.java

```diff
@@ -159,27 +184,57 @@ public void interceptAfterAllMethod(
       ExtensionContext extensionContext) {
     extractAccessIdAndPerformInvocation(invocation, invocationContext.getExecutable());
   }
+
   // endregion
 
   // region TestTemplateInvocationContextProvider
+
   @Override
   public boolean supportsTestTemplate(ExtensionContext context) {
-    return isAnnotated(context.getRequiredTestMethod(), WithAccessIds.class);
+    return isAnnotated(context.getElement(), WithAccessIds.class)
+        || isAnnotated(context.getElement(), WithAccessId.class);
   }
 
   @Override
   public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContexts(
       ExtensionContext context) {
-    WithAccessIds annotation = context.getRequiredTestMethod().getAnnotation(WithAccessIds.class);
+    List<WithAccessId> accessIds =
+        AnnotationSupport.findRepeatableAnnotations(context.getElement(), WithAccessId.class);
     Store store = getStore(context);
-    return Stream.of(annotation.value())
+    return accessIds.stream()
         .peek(a -> store.put(ACCESS_IDS_STORE_KEY, a))
         .map(JaasExtensionInvocationContext::new);
   }
+
   // endregion
 
+  private static void persistDynamicContainerChildren(
+      Iterable<DynamicNode> nodes, Map<String, List<DynamicNode>> map) {
+    nodes.forEach(
+        node -> {
+          if (node instanceof DynamicContainer) {
+            DynamicContainer container = (DynamicContainer) node;
+            List<DynamicNode> collect = container.getChildren().collect(Collectors.toList());
```

Comment:
could we use dynamicNodes or children instead of collect?

Reply:
done